bin/airgap-include.sh

```diff
@@ -0,0 +1,56 @@
+# Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This file is not marked as executable as it is intended to be sourced
+# Current directory must be the root directory of the repo
+
+## Check for AIRGAP_REGISTRY, if null/empty, error out.  Otherwise set and create HELM_URL_BASE.
```

Comment:
Take a look at [bin/common.sh|https://github.com/sassoftware/viya4-monitoring-kubernetes/blob/e542b813ea8c1d5126acd7b69401d84b2d3f1ffa/bin/common.sh#L35] and [bin/openshift-include.sh|https://github.com/sassoftware/viya4-monitoring-kubernetes/blob/main/bin/openshift-include.sh#L81] to see how we ensure we only execute the common code once; we should do something similar here.